### PR TITLE
fix: faq header shows and hides in preview (FTDC-1476)

### PR DESCRIPTION
**Pull request by inuwan**

### CODE REVIEW

File: src/templates/faq-preview.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import { useMemo } from "react";
+import {getHeadConfig as getFaqHeadConfig} from './faq'
+
+export const config: TemplateConfig = {
+    // The name of the feature. If not set the name of this file will be used (without extension).
+    // Use this when you need to override the feature name.
+    name: "FAQ Preview",
+};
+
+// The path must be exactly 404.html
```
Comment (bheus) [inline src/templates/faq-preview.tsx:23]: I think you copied this from somewhere else. Do we need to add a `.html` to the path?

File: src/hooks/useIsPreview.ts [hunk 2]
```diff
@@ -1,14 +1,7 @@
 import {useEffect, useMemo, useState} from "react";
 
 const useIsPreview = (): boolean => {
-  // Try to get an accurate value for preview at start to avoid showing page rendered
-  // as no-preview and then a second later change to preview
-  const defaultPreview = useMemo(() => {
```
Comment (bheus) [inline src/hooks/useIsPreview.ts:6]: Can we remove `useMemo` now that it's not being used?

File: src/templates/faq-preview.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                        </H4>
+                        <FaqList />
+                    </main>
+                </div>
+            </AnalyticsScreen>
+        </ConfigContext.Provider>
+    );
+};
+
+export default FAQPreview;
```
Comment (bheus) [inline src/templates/faq-preview.tsx:49]: Where is FAQPreview used?

**Review by bheus — CHANGES_REQUESTED** [review]

File: src/templates/faq-preview.tsx (hunk 3 above)
Comment (inuwan) [reply src/templates/faq-preview.tsx:49]: it will be used from the matching experience within a modal

File: src/hooks/useIsPreview.ts (hunk 2 above)
Comment (inuwan) [reply src/hooks/useIsPreview.ts:6]: PR updated

File: src/templates/faq-preview.tsx (hunk 1 above)
Comment (inuwan) [reply src/templates/faq-preview.tsx:23]: Don't think so - the "faq" page does not add because I thinks its a path and not a full address in this case

**Review by bheus — APPROVED** [review]

**Review by tdchristopher — APPROVED** [review]
looks good!

**Pull request merged**